packages/client/hmi-client/src/components/workflow/tera-simulate-ciemss.vue

```diff
@@ -218,7 +219,7 @@ const selectedCols = ref<string[]>([]);
 const paginatorRows = ref(10);
 const paginatorFirst = ref(0);
 const completedRunId = computed<string | undefined>(() => props?.node?.outputs?.[0]?.value?.[0]);
-const projectId = ref<string>(props.project.id);
+const projectId = ref(props?.project?.id);
```

Comment:
Why are `projectId`s  refs? I think it should be computed or even a normal variable

Reply:
I completely agree, we need to clean up all those workflow-node. Let me work on that on this PR, a bit more clean up shouldn't be an issue.